src/Components/CommentsWidget.tsx

```diff
@@ -226,8 +226,12 @@ export function CommentForm(props: CommentFormProps) {
                 ...(props.allowRating && rating ? { additional: { rating } } : {}),
             }),
         })
-            .then((res) => {
-                if (!res.ok) throw new Error('Unexpected response from comments server.');
+            .then(async (res) => {
+                const payload = await res.json();
+
+                if (payload.message?.includes('length must be less than')) {
```

Comment:
So we should do this instead?

```suggestion
                if (payload?.message?.includes('length must be less than')) {
```

Reply:
But `payload.message?` is already guarding against a payload potentially not having a message?

Your suggested change would guard against `payload` itself being `null` or `undefined`. If the backend is behaving correctly, we do always send a JSON payload with a `message` (https://github.com/datenanfragen/backend/blob/b_length-limits/src/handlers/comments/putComment.js#L11, https://github.com/datenanfragen/backend/blob/b_length-limits/src/handlers/comments/putComment.js#L32, https://github.com/datenanfragen/backend/blob/b_length-limits/src/handlers/comments/putComment.js#L36).  
The only way for `payload.message` to throw is if the response body was `null`. If that happens, the backend is clearly malfunctioning and we would drop into the `catch` anyway, which shows an appropriate error message.